### ISSUE TITLE
Chage Data Grid Timestamp format for Fb4 compatibility

### ISSUE THREAD

**Issue by arvanus** [mariuz/flamerobin#178]
The current format of timestamp is `D.N.Y, H:M:S.T`, but for Fb4 this format returns a exception in execution, so I'll remove the comma from the default format, new format:  `D.N.Y, H:M:S.T`. @mariuz any problem with it?
Read: https://sourceforge.net/p/firebird/mailman/firebird-devel/thread/24531455f1eba9738ec86f9de6c373be@lawinegevaar.nl/
Thanks

**Comment by luronumen** [comment]
My sugesttion is use **yyyy-MM-dd HH:mm:ss.t** for timestamp format.

Best Regards,
Luciano

**Comment by arvanus** [comment]
My concern is not to change a lot from the current format, because of this I suggest just to remove the comma

**Comment by luronumen** [comment]
You're right @arvanus ! In my opinion, we should focus first on full support for Firebird 3 and then think about support for Firebird 4.

Best Regards,
Luciano

**Comment by Jdochoa** [comment]
> You're right @arvanus ! In my opinion, we should focus first on full support for Firebird 3 and then think about support for Firebird 4.
> 
> Best Regards,
> Luciano
I agree

**Comment by mariuz** [comment]
The issue described in [#178](https://github.com/mariuz/flamerobin/issues/178) was that the default timestamp format `D.N.Y, H:M:S.T` included a comma which caused exceptions in Firebird 4 compatibility. The proposed fix was to remove the comma, resulting in the new default format: `D.N.Y H:M:S.T`.

I have reviewed the current state of the repository `mariuz/flamerobin` and can confirm that the fix is implemented in the codebase.

### Evidence of implementation

In `src/gui/controls/DataGridRows.cpp`, the default value for the `TimestampFormat` configuration has been updated to remove the comma:

```cpp name=src/gui/controls/DataGridRows.cpp url=https://github.com/mariuz/flamerobin/blob/cb0e84b6c3ce618dad9728ccc8e62034da7a33d3/src/gui/controls/DataGridRows.cpp#L74-L75
74|     timestampFormatM = config().get("TimestampFormat",
75|         wxString("D.N.Y H:M:S.T"));
```

Additionally, the `setFieldValue` method (lines 2550-2643) handles the conversion of fields back to Firebird-compatible strings. By ensuring the default format used for display and parsing matches a valid Firebird literal (or using the `getAsFirebirdString` method which uses the standard ISO-like `YYYY-MM-DD HH:MM:SS.MMM` format internally), the compatibility issue with Firebird 4 is resolved.

### Summary
The fix is **verified**. The comma was removed from the default configuration string in the `GridCellFormats::loadFromConfig()` method, which governs the default presentation and editing format for timestamps in the Data Grid.
